app/controllers/users_controller.rb

```diff
@@ -10,6 +10,12 @@ def destroy
     render json: "", status: 204
   end
 
+  def me
```

Comment:
this method should not be named `me`, the isn't relevant to what this method does at all. it should probably be named `update`